src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,425 @@
+The BIDS standard accepts microscopy data in different file formats to accommodate datasets
+stored in 2D image formats and whole-slide imaging formats, to accommodate lossless and lossy
+compression, and to avoid unnecessary conversions of the original data from a non-tiled to a
+tiled format, or vice-versa.
+
+For BIDS, Microscopy raw data MUST be stored in one of the following formats:
+
+-   [Portable Network Graphics](http://www.libpng.org/pub/png/) (`.png`)
+
+-   [Tag Image File Format](https://www.adobe.io/open/standards/TIFF.html) (`.tif`)
+
+-   [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#)
+    (`.ome.tif` for standard TIFF files or `.ome.btf` for
+    [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) files)
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats, for example with the
+Next-Generation File Formats currently developed by OME ([OME-NGFF](https://ngff.openmicroscopy.org/latest/))
+as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Name**                                      | `suffix` |
+| --------------------------------------------- | -------- |
+| Transmission electron microscopy              | TEM      |
+| Scanning electron microscopy                  | SEM      |
+| Micro-CT                                      | uCT      |
+| Bright-field microscopy                       | BF       |
+| Dark-field microscopy                         | DF       |
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
+| Coherent anti-Stokes Raman spectroscopy       | CARS     |
+| 2-photon excitation microscopy                | 2PE      |
+| Multi-photon excitation microscopy            | MPE      |
+| Super-resolution microscopy                   | SR       |
+| Nonlinear optical microscopy                  | NLO      |
+| Optical coherence tomography                  | OCT      |
+| Selective plane illumination microscopy       | SPIM     |
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
```

Comment:
Right now it might not be obvious (from the `[xx]` square brackets) that session is not required.  I suggest to add "Consistent with other data types in BIDS, the specification of the session is optional."

Reply:
Thank you, I integrate your suggestion.